canonical/store-components
# make each package card's height the same

## Done
- made each package card's height the same

### QA steps
- visit https://store-components-48.demos.haus/?path=/story/charmcard--lists and see if each card in different rows has the same height

## Fixes
Fixes: #https://warthogs.atlassian.net/browse/WD-4304
